src/jit/compiler.h

```diff
@@ -487,6 +487,45 @@ class LclVarDsc
         assert(_lvRegNum == reg);
     }
 
+    const size_t lvArgStackSize() const
+    {
```

Comment:
There are a few other lva* functions in compiler.hpp. You could put it there. Or in lclvars.cpp.